pythoncalculator/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .add import add 
+from .multiply import multiply 
```

Comment:
Please add the add function

```suggestion
from .add import add
from .multiply import multiply 
```